src/Tests/Logic/StrippableTextTest.cs

```diff
@@ -57,9 +57,9 @@ public void StrippableTextItalic3()
         public void StrippableTextFontDontTouch()
         {
             var st = new StrippableText("{MAN} Hi, how are you today!");
-            Assert.AreEqual("", st.Pre);
+            Assert.AreEqual("{", st.Pre);
```

Comment:
I think this test was wrong.

We should treat '{' as '[' When it hold hearing impaired word/sentence